source/index.html.md.erb

```diff
@@ -1,18 +1,27 @@
-# Your digital workplace
-
-<%= partial "partials/menu" %>
-
-## A new intranet
-
-The digital workplace project aims to give everyone at Citizens Advice
-easy access to the infromation, knowledge, and tools they need to do
-their jobs. As part of this, we're building a new Intranet to replace
-CABlink and BMIS. When it's live, you'll be able to access the new
-intranet from this page.
-
-In the meantime, you can explore what will live on the new intranet and
-what should be on another platform or channel.
-
-<a href="https://www.citizensadvice.org.uk/EPiServer/CMS/Home" class="cads-button cads-button__primary">
-  Find out where will content live
-</a>
+<div class="cads-page-content">
+  <div class="cads-grid-container">
+    <div class="cads-grid-row">
+      <main class="cads-grid-col-12">
+        <h1 class="cads-page-title">Your digital workplace</h1>
+        <%= partial "partials/menu" %>
+        <div class="cads-prose">
+          <h2>A new intranet</h2>
+          <p>
```

Comment:
FYI you can change the filename back to `.html.erb` if we're not using markdown here anymore, if you want